src/main/java/dev/jbang/net/JdkManager.java

```diff
@@ -88,6 +88,17 @@ public static Path downloadAndInstallJdk(int version) {
 		}
 	}
 
+	public static String resolveInJavaHome(String cmd, String requestedVersion) {
```

Comment:
I think this is best moved to `JavaUtil`. The `JdkManager` really only handles install/uninstall of JDKs and telling you where they can be found.

Reply:
moved.